Neos.Flow/Classes/Command/ResourceCommandController.php

```diff
@@ -161,7 +161,7 @@ public function copyCommand(string $sourceCollection, string $targetCollection,
             $this->quit(1);
         }
 
-        if (!empty($targetCollection->getObjects())) {
+        if (!$targetCollection->getObjects()) {
             $this->outputLine('The target collection "%s" is not empty.', [$targetCollectionName]);
             $this->quit(1);
         }
```

Comment:
Just checked again how generators behave with a simple truthy check:
https://3v4l.org/sk4P8#v7.4.21

So this whole check is effectively `if (false) {` now, which doesn't make much sense either. Looking again at the docs and checking behaviour of `Generator::valid()` this seems to be the correct way:
```suggestion
        if ($targetCollection->getObjects()->valid()) {
            $this->outputLine('The target collection "%s" is not empty.', [$targetCollectionName]);
            $this->quit(1);
        }
```
https://3v4l.org/UGZ2I#v7.4.21